### DISCUSSION TITLE
Videohub Maximum Size

### DISCUSSION THREAD

**Discussion by nearlybond** [bitfocus/companion#2533] [Q&A] [answered]
I was wondering if it were possible to increase the size of the videohub module to be more than the current 288? I am using the protocol to control something with 512 inputs and outputs, and the BlackMagic software can see and control those inputs above 288.

**Comment by Julusian** [comment, marked as answer]
This would be better to post in the repository for that module https://github.com/bitfocus/companion-module-bmd-videohub

But yes, it being limited to 288 is most likely an artificial limitation as that is all that is needed for the official hardware

**Reply by nearlybond** [reply]
Thank you!
